app/src/main/res/values/strings.xml

```diff
@@ -128,6 +128,9 @@
     <!-- Preference category: Custom (user-defined) autocomplete URLs. -->
     <string name="preference_category_autocomplete_custom">Custom URL</string>
```

Comment:
Can we remove this one now?

Reply:
I tried to remove all declarations but it formatted it... Do we need to edit all files or does removing it from the main strings.xml remove it from our translations?

Reply:
IIRC removing it from the main `strings.xml` will remove it from our translations on the next export/import

Reply:
Had lint errors, so going to open a new issue to address unused strings

Reply:
Or I can take it out and we can override the lint errors?

Reply:
I usually remove the string from strings.xml and then run export/import again - That will remove it from the translations.